docs/modules/ROOT/pages/advanced-how-tos/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,61 @@
+= Maintaining remote Tekton pipelines up to date
+
+== The problem
+
+Although Tekton remote pipelines is a huge improvement for having a centralized and shared repo for Pipelines and Tasks, it comes with a drawback; the Pipeline CRDs will not be stored in the Konflux Component repository anymore, so MintMaker will not be able to update the TaskRef digests for us.
```

Comment:
I feel like this problem is slightly incomplete. Is the issue that you are trying to describe here one where a common pipeline is self-managed based on Konflux tasks? Are you creating a self-defined "component template" which is then used as the source for the pipeline definition for other components?

If so, how should the other components reference the pipelines and tasks that are built here?

Reply:
mmh good point, I don't have a strong opinion on where this should be placed, I'm okay with that. Should I also emphasize the advantages of using remote pipelines? Like three different sections: the pros (and how to use remote pipelines), the drawbacks, the solution to the drawback.

Reply:
Yes, I think that sounds great! You can use the existing workflow as a reference for some of the content that we included before. We can also generalize the workflows section (i.e. title) as it is a little overly specific to the git submodule approach.

Reply:
Moved to workflows,added advantages section and ready for the next round of reviews :)